playbook/app/pb_kits/playbook/pb_date_time_stacked/date_time_stacked.test.js

```diff
@@ -2,10 +2,19 @@ import { ensureAccessible, renderKit } from '../utilities/test-utils'
 import { DateTimeStacked } from '../'
 
 /* eslint-disable jsx-control-statements/jsx-jcs-no-undef */
+const currentDate = new Date()
+
+const datetime = new Date('Wed Mar 31 2021 12:00:00 GMT-0500'),
+  monthName = datetime.toLocaleString('en-US', { month: 'short' }),
+  day = datetime.getDate(),
+  fullYear = datetime.getFullYear(),
+  optionalYear = currentDate.getFullYear() !== fullYear ? fullYear : ''
+
+const monthDayYear = `${monthName}${day}${optionalYear}`
```

Comment:
Since we just rolled on to 2022 (a new year) you will notice that we have logic in `DateTimeStacked` to show the year if it is not the current year. This means the old spec which passed `datetime` as `new Date('Wed Mar 31 2021 12:00:00 GMT-0500')` would fail exactly on New Years while we drink our Champagne (kombucha for me this year 🤣 )

Reply:
🤣🤣🤣

Reply:
More like Kom-gotcha.